addons/point_of_sale/models/pos_session.py

```diff
@@ -1945,16 +1945,10 @@ class ProcurementGroup(models.Model):
 
     @api.model
     def _run_scheduler_tasks(self, use_new_cursor=False, company_id=False):
-        super(ProcurementGroup, self)._run_scheduler_tasks(use_new_cursor=use_new_cursor, company_id=company_id)
+        super()._run_scheduler_tasks(use_new_cursor=use_new_cursor, company_id=company_id)
         self.env['pos.session']._alert_old_session()
-        if 'scheduler_task_done' in self._context:
-            task_done = self._context.get('scheduler_task_done', {'task_done': 0})['task_done'] + 1
-            self._context['scheduler_task_done']['task_done'] = task_done
```

Comment:
do you know what this context was? it doesn't seem used
cc @Whenrow

Reply:
It is completely unused anywhere and very counter-intuitive.
There is just a commit message "They will be split in 5 different crons later to be able to count" in 5f6e1ed8c81.